icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -291,6 +254,22 @@ ant check 2>&1 | tee /tmp/icu4j-newData-antCheck.txt
 # more failures in ICU4C or ICU4J (except failures that were present before you
 # began testing the new CLDR data).
 
+# Note that certain data changes and related test failures may require the
+# rebuilding of other kinds of data. For example:
+# a) Changes to locale matching data may cause failures in e.g. the following:
+#      com.ibm.icu.dev.test.util.LocaleDistanceTest (testLoadedDataSameAsBuiltFromScratch)
+#      com.ibm.icu.dev.test.util.LocaleMatcherTest (testLikelySubtagsLoadedDataSameAsBuiltFromScratch0
```

Comment:
*If* you are making other changes, please replace the trailing '0' with ')'.

Reply:
Done